docs/cheatsheet/annotations.rst

```diff
@@ -0,0 +1,16 @@
+.. _ref_cheatsheet_annotations:
+
+Annotations
+===========
+
+Flag all reviews to a specific movie:
+
+.. code-block:: sdl
+
+    type Label {
+        annotation title :=
+            'Special label for some \
+            choice reviews';
+        required property comments -> str;
+        link review -> Review;
+    }
```

Comment:
Add an anno for the link too. And add an example of introspecting annos

Reply:
OK